### DISCUSSION TITLE
Need United Kingdom version of Synthea data

### DISCUSSION THREAD

**Discussion by tariqsherwani1** [synthetichealth/synthea#1209] [Q&A]
Hi,

Does anyone know how I can translate the Synthea CSV files to reflect UK health/demographics? I need to somehow maintain the relationships between variables such as healthcare providers, ethnicity, gender, socioencomic conditions, religion, etc. This is because ML models will be built on top of this data, hence I am not able to create it from scratch in the short amount of time I have.

Any help would be appreciated.

Thanks!

**Comment by jawalonoski** [comment]
Did you look at the Synthea-International repository?

https://github.com/synthetichealth/synthea-international

Try the `gb` folder which should include England, Scotland, and Wales.

**Reply by tariqsherwani1** [reply]
Thank you for that.

Is there a way I can map, for example, if someone lives in New York to an equivalent city in the UK? I need to maintain the socioeconomic logic embedded in the Synthea dataset. Essentially I need to map US Zip Codes to UK post codes in order to maintain that socioeconomic factor. I also need to model the ethnicty in the US dataset to fit the demographics of the UK. Do you have any ideas how I can do that?

Also do you know where I can find the data model for the CSV Synthea files?

Thanks

**Reply by jawalonoski** [reply]
The `gb/src/main/resources/geography/demographics.csv` contains some of the socioeconomic factors described [here](https://github.com/synthetichealth/synthea/wiki/Demographics-for-Other-Areas#demographics-file-format), although admittedly all the locations have the same values (it looks like the file is using national level statistics).

I would not map US locations to UK locations, and I would strongly recommend not doing that except for the most trivial experiments... that seems like a methodology full of error.

Alternatively, I would search for public demographic and economic data from whatever UK organization(s) are equivalent to the US Census Bureau and just build a proper file.

**Reply by tariqsherwani1** [reply]
Thanks will try. Is there a way I can show you what my plan is and what I am trying to achieve?

**Reply by jawalonoski** [reply]
> Is there a way I can show you what my plan is and what I am trying to achieve?

Well, you could post it here, or you could write to us at synthea-list@groups.mitre.org.

**Reply by tariqsherwani1** [reply]
Hi,

Thanks for the reply. Can you please tell me if there is a way I can import the source code into a Jupyter notebook and recreate the 13 CSV files for an x number of patients? I need to recreate the files for around 500k patients.

Thanks

**Comment by lhs-open** [comment]
Your original question of mapping US data to UK data is probably more difficult than doing the way Synthea suggests. No one knows  how to map them because the mapping rules don't exist. If you have the statistics of UK healthcare, it would be more practical to feed the data to Synthea, as Jay and Synthea suggest above.  The results would be reusable for the global community.
